install/geoip.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if [ ! -f 'install.sh' ]; then echo 'Where are you?'; exit 1; fi
+
+dc="docker-compose --no-ansi"
+dcr="$dc run --rm"
```

Comment:
These all suggest me that this file should not be a separate module. Or this file needs to be `source`d. I still prefer things being in a single place until we have a very good reason tho.

Reply:
"These all" ... there are only two. :P

I agree there is more refactoring to be done, I didn't want to load this PR up too much or block it on a bigger refactor.

Reply:
Let's do something like the following:

- https://stackoverflow.com/a/42101141/90297
- https://stackoverflow.com/a/10822868/90297
- https://stackoverflow.com/a/36830054/90297